content/collections/audit-works.html

```diff
@@ -1,3 +1,195 @@
+  </field>
+<field name="subtitle">
+   A protocol for decentralized social apps
+  </field>
+  <field name="date">
+    2023-08-29
+  </field>
+  <field name="about">
+    Audit of two new features in the `KeyRegistry` and `IdRegistry` contracts.
+  </field>
+  <field name="report">
+    /library/audits/farcaster-2
+  </field>
+  <field name="attachments">
+    [Repository](https://github.com/farcasterxyz/contracts)
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    farcaster.webp
+  </field>
+  <field name="title">
+    Farcaster-1
+  </field>
+<field name="subtitle">
+   A protocol for decentralized social apps
+  </field>
+  <field name="date">
+    2023-08-18
+  </field>
+  <field name="about">
+    Audit of Farcaster's core L1 and L2 contracts.
+  </field>
+  <field name="report">
+    /library/audits/farcaster-1
+  </field>
+  <field name="attachments">
+    [Repository](https://github.com/farcasterxyz/contracts)
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    pooltogether.png
+  </field>
+  <field name="title">
+    PoolTogether-A-1
+  </field>
+  <field name="subtitle">
+    A decentralized prize savings protocol
+  </field>
+  <field name="date">
+    2023-9-19
+  </field>
+  <field name="about">
+    Audited V5 Prize Pool and V5 TWAB Controller contracts.
+  </field>
+  <field name="report">
+    /library/audits/pooltogether-1
+  </field>
+  <field name="attachments">
+    [Github Organization](https://github.com/pooltogether)
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    connext.png
+  </field>
+  <field name="title">
+    Connext-A-4
+  </field>
+  <field name="subtitle">
+    A modular interoperability protocol
+  </field>
+  <field name="date">
+    2023-10-18
+  </field>
+  <field name="about">
+    Audited upgrades to support an optimistic system on Spoke Connectors.
+  </field>
+  <field name="report">
+    /library/audits/connext-4
+  </field>
+  <field name="attachments">
+    [Repository](https://github.com/connext/monorepo)
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    connext.png
+  </field>
+  <field name="title">
+    Connext-A-3
+  </field>
+  <field name="subtitle">
+    A modular interoperability protocol
+  </field>
+  <field name="date">
+    2023-07-6
+  </field>
+  <field name="about">
+   Audit of Wormhole hub and spoke connectors.
+  </field>
+  <field name="report">
+    /library/audits/connext-3
+  </field>
+  <field name="attachments">
+    [Repository](https://github.com/connext/monorepo)
+  </field>
+</item>
+
+<item>
+  <field name="logo">
+    connext.png
+  </field>
+  <field name="title">
+    Connext-A-2
+  </field>
+  <field name="subtitle">
+    A modular interoperability protocol
+  </field>
+  <field name="date">
+    2023-07-12
+  </field>
+  <field name="about">
+    Audit scope included, among other things, new connectors to support additional chains, templatized `IXReceiver` contracts, and updates to the Optimism connector to support Bedrock.
```

Comment:
We should also add: "and adding Optimistic roots initial implementation to avoid the direct use of AMBs to propagate messages to the RootManager" 
Maybe too long, but the optimistic root was one main part of the audit

Reply:
I don't think it's too long :) Adding now!